src/Sentry/SentryOptionsExtensions.cs

```diff
@@ -32,6 +32,13 @@ public static void DisableDuplicateEventDetection(this SentryOptions options)
         public static void DisableAppDomainUnhandledExceptionCapture(this SentryOptions options)
             => options.Integrations = options.Integrations.RemoveAll(p => p.GetType() == typeof(AppDomainUnhandledExceptionIntegration));
 
+        /// <summary>
+        /// Disables the capture of errors through <see cref="AppDomain.UnhandledException"/>
+        /// </summary>
+        /// <param name="options">The SentryOptions to remove the integration from.</param>
+        public static void DisableAppDomainProcessExitFlush(this SentryOptions options)
+            => options.Integrations = options.Integrations.RemoveAll(p => p.GetType() == typeof(AppDomainProcessExitIntegration));
```

Comment:
Now that we have two, we could create an internal extension method to `IEnumerable<ISdkIntegration>` called `RemoveIntegration<T>`:

```csharp
internal static IImmutableList RemoveIntegration<TIntegratiom>(this IImmutableList<ISdkIntegration> integrations);
=> integrations.RemoveAll(p => p.GetType() == typeof(TIntegratiom));
```